components/campaign-hero/Component.jsx

```diff
@@ -0,0 +1,243 @@
+  const [isUserPaused, setIsUserPaused] = useState(false); // Track if the video was paused by the user
+  const [isPlaying, setIsPlaying] = useState(false);
+  const iframeRef = useRef(null);
+
+  const handleClick = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
+
+  // Toggles play/pause when button is clicked
+  const togglePlayPause = () => {
+    setIsUserPaused((prev) => !prev); // Track manual pause state
+    setIsPlaying((prev) => !prev); // Toggle play/pause state
+  };
+
+  // Send postMessage to Vimeo iframe to play/pause video
+  useEffect(() => {
+    const iframe = iframeRef.current;
+    if (iframe && iframe.contentWindow) {
+      iframe.contentWindow.postMessage(
+        JSON.stringify({ method: isPlaying ? "play" : "pause" }),
+        "*"
+      );
+    }
+  }, [isPlaying]);
+
+  // Pause video when it's out of viewport using IntersectionObserver
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      (entries) => {
+        const [entry] = entries;
+
+        // Only set play/pause based on IntersectionObserver if video is not manually paused
+        if (!isUserPaused) {
+          setIsPlaying(entry.isIntersecting);
+        }
+      },
+      { threshold: 0 } // Adjust threshold as needed
+    );
+
+    const iframe = iframeRef.current;
+    if (iframe) observer.observe(iframe);
+
+    return () => {
+      if (iframe) observer.unobserve(iframe);
+    };
+  }, [isUserPaused]); // Re-run the effect if isUserPaused state changes
+
+  return (
+    <Container width="full" paddingX={false} className={styles.root}>
+      <section className={styles.section}>
+        <div className={styles.heroWrapper}>
+          {/* Background video or image */}
+          {isBgVideo ? (
+            <div className={styles.bgWrapper}>
+              <div className={styles.bgVideoWrapper}>
+                <iframe
+                  ref={iframeRef}
+                  src={`${bkgConfig.bkgVideo}?background=1`}
+                  title="background video"
+                  className={styles.bgVideoIframe}
+                  allow="autoplay; fullscreen"
+                  allowFullScreen
+                />
+              </div>
+            </div>
+          ) : (
+            <div className={styles.bgImageWrapper}>
+              <img
+                src={bkgImageData?.url}
+                className={styles.bgImage}
+                alt={bkgImageData?.attributes?.alt || ""}
+              />
+            </div>
+          )}
+          {/* Dark overlays */}
+          <div
+            className={styles.overlay(hasQuote, isBgVideo)}
+            aria-hidden="true"
+          />
+          {/* Hero content */}
+          <div
+            className={styles.contentWrapper(
+              hasQuote,
+              isBgVideo,
+              isIntroPulledLeft
+            )}
+          >
+            <h2 className={styles.title}>{textConfig.title}</h2>
```

Comment:
Justin, I believe you originally had this as a h1, but I found that there's already a screen reader only h1 (same as the page name) even when it's not visible, so I change the hero title to an h2 instead.

Reply:
Hmmm. I need to look back at Matrix to see what the conditions for this `sr-only` H1 are and discuss with the team. I do see the `sr-only` H1 in the markup but I don't think we want that here... If we end up with an H1 that is identical to the H2 then it will be repetitive and an different a11y error to resolve. If the editors title the page they create something only meaningful for administrative purposes we also probably don't want that read out to screen reader users.

Reply:
@iamrentman Good point. I wonder if there's a way to hide the SR only H1 as well, but yeah the most ideal way would be to use an H1 in the hero and hide this SR only one.

Reply:
I've added logic to the Matrix templates so that the `upper_content_include` which was printing the `sr-only` H1 does not print at all on Campaign pages.

I have also updated the headline here in the Component.jsx file to an H1.

Reply:
Oh awesome - thank you for looking into that @iamrentman!